README.md

```diff
@@ -1,2 +1,141 @@
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+
+  <h3><b>Hello Microverse</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+-   [📖 About the Project](#about-project)
+    -   [🛠 Built With](#built-with)
+        -   [Tech Stack](#tech-stack)
+-   [💻 Getting Started](#getting-started)
+    -   [Setup](#setup)
+    -   [Prerequisites](#prerequisites)
+    -   [Install](#install)
+    -   [Usage](#usage)
+-   [👥 Authors](#authors)
+-   [🤝 Contributing](#contributing)
+-   [⭐️ Show your support](#support)
+-   [❓ FAQ (OPTIONAL)](#faq)
+-   [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Hello_Microverse <a name="about-project"></a>
+
+> This is my first project at microverse to learn how to make project, add files and main thing how to setup linter in a project.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML</summary>
+</details>
+
+<details>
+  <summary>CSS</summary>
+</details>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+setup linter
+open this link: https://github.com/microverseinc/linters-config/tree/master/html-css
+copy this folder to your project at root directory
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+cd my-folder
+git clone git@github.com:myaccount/my-project.git
+
+### Usage
+
+To run the project, execute the following command:
+
+To run tests, run the following command:
+install liveserver extension
+right click on index.html and click on live server
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Shah Zaman**
+
+-   GitHub: [@sheezu900](https://github.com/sheezu900)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/sheezu900/Hello-world/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give a star to this repository
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT] licensed.
+
```

Comment:
{OPTIONAL}
- [ ] Kindly fix your LICENSE section as no link is found and neither a LICENSE file is listed in your project's files.

---

- _Please refer to this simple [tutorial](https://www.youtube.com/watch?v=8Dm7XXvtqXI) which will help you create one, please do not forget to choose the "MIT" license._